src/configuraton.ts

```diff
@@ -50,6 +52,11 @@ export class Configuration {
     trustedProxies = [],
     trustProxyChain = false,
     trustedProxyDepth = 0,
+    logger = pino({
```

Comment:
Can we remove pino from the prod parts of the code ?

Reply:
Unfortunately, we need to temporarily continue development with pino, because changing the logger requires more complex changes in `LoggerService`. We have the task for a configurable logger in the backlog